cmd/aries-agent-mobile/go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/pquerna/cachecontrol v0.1.0 // indirect
 	github.com/rs/cors v1.7.0 // indirect
+	github.com/santhosh-tekuri/jsonschema v1.2.4 // indirect
```

Comment:
is `xeipuuv/gojsonschema` package still referenced somewhere? I see indirect references at lines 60-62 below.

It would be good to remove all references to `xeipuuv` packages

Reply:
@Baha-sk - @Abdulbois wanted to show one example of the change to gather feedback. Once there is some feedback, all instances will be changed.

Reply:
I am for this change. A couple of days ago the old package was failing bbs signature post signing when validating the json map where an embedded json struct was being wrongly resolving to an empty RDF entry. Hopefully, this package does a better job at mapping jsonld docs.